app/controllers/api/braintree_controller.rb

```diff
@@ -37,11 +38,15 @@ def subscription_options
     {
       amount: params[:amount].to_f,
       plan_id: '35wm',
-      payment_method_token: Payment.customer(params[:email]).default_payment_method.token
+      payment_method_token: default_payment_method_token
     }
   end
 
   def braintree
     PaymentProcessor::Clients::Braintree
   end
+
+  def default_payment_method_token
+    @token ||= ::Payment.customer(params[:email]).try(:card_vault_token)
```

Comment:
With `@token ||= ::Payment.customer(params[:email]).try(:card_vault_token)`, what happens if the customer has no `card_vault_token` and nil gets passed as `payment_method_token`? Will this raise on subscription so that we send the error message to the client?